src/content/docs/synthetics/synthetic-monitoring/using-monitors/store-secure-credentials-scripted-browsers-api-tests.mdx

```diff
@@ -191,9 +191,9 @@ We currently redact the following from the results of your monitor:
 
 When creating a script, ensure that secure credentials are not passed to an external site or entered into a website in a way that causes them to be displayed or otherwise disclosed. 
 
-The protection mechanisms detailed above are designed to prevent disclosure of the credentials when stored by New Relic and in-text responses from the website being monitored. However, the scripting capability allows users to implement functionality that could result in the disclosure of the credentials if misused by the user. New Relic does not control the site being monitored or the actions of individual users of your organization. For this reason, you should only grant permission to use secure credentials in scripts to users that are fully trusted and authorized to do so by your organization. To update permissions for secure credentials in Synthetics monitoring, please follow [user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts) to control [capabilities for secure credentials](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#synthetics). 
+The protection mechanisms detailed above are designed to prevent disclosure of the credentials when stored by New Relic and in-text responses from the website being monitored. However, the scripting capability allows users to implement functionality that could result in the disclosure of the credentials if misused by the user. New Relic does not control the site being monitored or the actions of individual users of your organization. For this reason, you should only grant the "Use credentials" permission to users that are fully trusted and authorized to do so by your organization. To update permissions for secure credentials in Synthetics monitoring, please follow [user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts) to control [capabilities for secure credentials](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#synthetics). 
```

Comment:
```suggestion
The protection mechanisms detailed above are designed to prevent disclosure of the credentials when stored by New Relic and in-text responses from the website being monitored. However, the scripting capability allows users to implement functionality that could result in the disclosure of the credentials if misused by the user. New Relic does not control the site being monitored or the actions of individual users of your organization. For this reason, you should only grant the "Use credentials" permission to users that are fully trusted and authorized to do so by your organization. To update permissions for secure credentials in Synthetics monitoring, please follow [user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts) to control [capabilities for secure credentials](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#synthetics). 
```